### PR TITLE
Added last_login and last_mobile

### PR DESCRIPTION
**Pull request by MineTech13**
I added last_login to LimitedUser.yaml and last_mobile to User.yaml and tested it for functionality ^^

### CODE REVIEW

File: openapi/components/schemas/User.yaml [hunk 1]
```diff
@@ -52,6 +52,10 @@ properties:
     description: Either a date-time or empty string.
   last_platform:
     $ref: ./Platform.yaml
+  last_mobile:
+    type: string
+    format: date-time
+    description: Either a date-time or empty string.
```
Comment (C0D3-M4513R) [inline openapi/components/schemas/User.yaml:58]: `last_mobile` is not present in the response:
https://github.com/vrchatapi/specification-test/blob/main/data/requests/users/getuser.md

File: openapi/components/schemas/LimitedUser.yaml [hunk 2]
```diff
@@ -51,6 +51,9 @@ properties:
     type: string
   friendKey:
     type: string
+  last_login:
+    type: string
+    format: date-time
```
Comment (C0D3-M4513R) [inline openapi/components/schemas/LimitedUser.yaml:56]: `last_login` is not present in the response of:
https://vrchat.com/api/1/users?search=tupper

**Review by C0D3-M4513R — CHANGES_REQUESTED** [review]

File: openapi/components/schemas/User.yaml (hunk 1 above)
Comment (MineTech13) [reply openapi/components/schemas/User.yaml:58]: Upon further investigation, I discovered and successfully tested that the field is only accessible if the authenticated account is friends with the target account.

File: openapi/components/schemas/LimitedUser.yaml (hunk 2 above)
Comment (MineTech13) [reply openapi/components/schemas/LimitedUser.yaml:56]: Upon further investigation, I discovered and successfully tested that the field is only accessible if the authenticated account is friends with the target account.

**Comment by C0D3-M4513R** [comment]
The fields should then be nullable, right?

**Comment by MineTech13** [comment]
I believe they can be nullable, as they only appear when a friendship is established.

**Comment by VinyarionHyarmendacil** [comment]
Strictly speaking, `nullable` means that the JSON `null` is a valid value and `required` means that the key-value pair must be present, e.g.:
| | `nullable` | `required` | both | neither |
|--------|--------|--------|--------|--------|
| `{ }` | ✔️ | ❌ | ❌ | ✔️ | 
| `{ "key": null }` | ✔️ | ❌ | ✔️| ❌ | 
| `{ "key": "value" }` | ✔️ | ✔️ | ✔️ | ✔️ |

**Comment by MineTech13** [comment]
In that case, it is indeed nullable, as it is only present when "isFriended" is true.

**Pull request closed without merging**